sha3_api.c

```diff
@@ -15,6 +16,12 @@ void sha3_init(sha3_var_t *c, size_t md_sz)
   size_t i;
 
   for (i = 0; i < 25; i++)
```

Comment:
Actually you can replace the entire loop by a memset. CBMC will understand that out of the box.

Reply:
Some devs like to avoid stdlib functions as in some environments they are not available.
The approach we have been taking in mlkem-native and mldsa-native is to use a wrapper around the stdlib functions (basically just memset, memcpy) and then provide a configuration option that allows providing a custom implementation for those (which could be a plain loop).